src/modules/webpack-publish.test.ts

```diff
@@ -0,0 +1,123 @@
+import { WebpackRestoreOptions } from "../interfaces/webpack-restore-options";
+import { NodeCI } from "./node-ci";
+import { NodeClean } from "./node-clean";
+import { NodeRestore } from "./node-restore";
+import { WebpackPublish } from "./webpack-publish";
+
+describe("webpack-publish", () => {
+    describe("restore", () => {
+        beforeEach(() => {
+            jest.spyOn(NodeClean, "run").mockImplementation();
+            jest.spyOn(NodeClean, "run").mockImplementation();
```

Comment:
Looks like the `NodeClean:run` spy was setup twice. If intentional (for some reason), please note why